Add `GBC 2022 Spring` Tournament Information (#7093)
ppy/osu-wiki: wiki/Tournaments/GBC/GBC_2022_Spring/en.md

---
tags:
  - GBC
  - GBC 2022
  - GBC2022
  - GBC 2022 Spring
---

# GB Cup 2022 Spring

> The **GB Cup 2022 Spring** (***GBC 4K 2021***) was a 3v3 captain draft-based osu!mania 4-key tournament hosted by the ![][flag_CN] [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407).

Reviewer: ```suggestion
The **GB Cup 2022 Spring** (***GBC 4K 2021***) was a 3v3 captain draft-based osu!mania 4-key tournament hosted by ![][flag_CN] [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407).
```

## Tournament schedule

| Event | Timestamp |
| --: | :-- |

> | Registration phase | 2022-03-26/2022-04-02 (23:59 UTC + 8) |
> | Draft phase | 2022-04-03 (21:00 UTC + 8) |

Reviewer: ```suggestion
| Registration phase | 2022-03-26/2022-04-02 (23:59 UTC+8) |
| Draft phase | 2022-04-03 (21:00 UTC+8) |
```

| Regular Stage Week 1 | 2022-04-09/2022-04-10 |
| Regular Stage Week 2 | 2022-04-16/2022-04-17 |
| Regular Stage Week 3 | 2022-04-23/2022-04-24 |
| Playoffs Week 1 | 2022-04-30/2022-05-01 |
| Playoffs Week 2 | 2022-05-07/2022-05-08 |

## Prizes

| Placing | Prizes |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | profile badge, KanonBot profile badge, 50 CNY ￥ per team member |

Reviewer: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, KanonBot profile badge, 50 CNY ￥ per team member |
```

Reviewer: additionally, do you really need to use the yuan symbol here (and in other places), since CNY already includes the word yuan?

Author: in fact, I don't know the concurrency rules on this wiki, on liquipedia it will use `¥ CNY`, since "¥" will also be used for JPN yuan.

| Referee | *TBA* |
| Commentator | ![][flag_CN] [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407) ![][flag_CN] [aiyulu](https://osu.ppy.sh/users/189617) ![][flag_CN] [\[GB\]AelSan](https://osu.ppy.sh/users/14095291) ![][flag_CN] [Edward_Tsui](https://osu.ppy.sh/users/8609627) |
| Designer | *TBA* |
| Statistician | ![][flag_CN] [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407)，![][flag_CN] [Rush_FTK](https://osu.ppy.sh/users/3046856)  |

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1546120)
- Livestream
- Challonge bracket
- Master sheet
- Statistics sheet

## Ruleset

### Tournament rules

1. The GB Cup 2022 Spring is a captain draft-based team tournament, played on the osu!mania game mode.
   - **While this competition is planned as a 3 versus 3 setup, this might change depending on the number of incoming registrations.**
2. Beatmap scoring will be based on Score V2.
3. The beatmaps for each round will be announced by the Mappool Selector Team on the Sunday before the actual matches take place.

>    - Each mappool will contain a *tiebreaker* beatmap. It will only be played in case of a tie in the scoreline in the *best of* system, e.g. the scoreline is 4-4 and the match is best of 9, the tiebreaker shall be played.

Reviewer: ```suggestion
   - Each mappool will contain a *tiebreaker* beatmap. It will only be played in case of a tie in the scoreline in the *best of* system, e.g. if the scoreline is 4-4 and the match is best of 9, the tiebreaker shall be played.
```

Author: this rule's description is a copy from MWC 4k 2021, so I thought it might right.

4. Match schedule will be handled by the Host.
5. If no referees are available at match time, the match will be postponed.
6. Failed players' scores do not get added to the team score.
   - Reviving and surviving during a beatmap is considered as passing it.

> 7. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins are allowed.

Reviewer: ```suggestion
7. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins is allowed.
```

Author: same as above, this rule's description is a copy from MWC 4k 2021, so I thought it might right.

18. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms, too.
    - Breaking the chat rules results in a silence. Silenced players cannot participate at multiplayer matches and must be exchanged for the time being.
19. Unexpected incidences will be handled by the Host. Referees may allow higher tolerance depending on the given circumstances. This is up to their discretion.
20. Penalties for violating the tournament rules include, but are not limited to:
    - Exclusion of specific players for one beatmap.
    - Exclusion of specific players for an entire match.
    - Declaring the match as forfeited, or as a win by default for the other team.
    - Disqualification from the entire tournament.
    - Disqualification from the current and future GBC series.
    - Disqualification from the current and future official & commnunity tournaments, until appealed.
21. The Host reserves the right to modify these rules at any moment. Any such canges will be announced in advance.

### Tournament registration

> 1. Please **send Host private messages** **and** filled [**this form**](https://wj.qq.com/s2/9902248/95cf/) to registration. 

Reviewer: ```suggestion
1. Please send the Host **a private message** and **fill [this form](https://wj.qq.com/s2/9902248/95cf/)** to register. 
```

2. Each player could send a VOD clips or an in-game replays to Host, which will be used on Draft phase. 
3. To ensure valid and serious registrations, every registered user will be manually checked by the Host.

>    - To be successfully registrations, player have not violated the [osu! community rules](/wiki/Rules) within the last 12 months.

Reviewer: ```suggestion
   - To be successfully registered, a player must not have violated the [osu! community rules](/wiki/Rules) within the last 12 months.
```

>    - **No rank limitations for this tournament.** but host have the rights to choose several **best** players may invited to **pricipate showmatch instead of actual tournament** for balance issues.

Reviewer: I don't understand this sentence, what were you trying to explain?

Author: no lowest/highest rank limitation for players. if amounts of registered players are higher than tournament limits, some top players (not fully equal to have highest PP or world ranking) will be removed from the player list.  
but this tournament has show matches between each stage, these players removed by above reason will be invited into such show match.

> 4. All successfully registrated player will be published after the Registration Phase.

Reviewer: ```suggestion
4. A list of all successfully registered players will be published after the Registration Phase.
```
